src/segments/project.go

```diff
@@ -211,7 +211,16 @@ func (n *Project) getDotnetProject(_ ProjectItem) *ProjectData {
 	// get the first match only
 	for _, file := range files {
 		extension := filepath.Ext(file.Name())
-		if extension == ".csproj" || extension == ".fsproj" || extension == ".vbproj" {
+		if extension == ".sln" || extension == ".slnf" {
+			name = strings.TrimSuffix(file.Name(), filepath.Ext(file.Name()))
+			content = n.env.FileContent(file.Name())
+
+			return &ProjectData{
+				Target: "",
+				Name:   name,
+			}
+		}
+		if extension == ".sln" || extension == ".slnf" || extension == ".csproj" || extension == ".fsproj" || extension == ".vbproj" {
```

Comment:
please refactor this to use an array and `Contains` from [slices](https://pkg.go.dev/slices#Contains).

Reply:
@JanDeDobbeleer, I'm not a go programmer, so I just followed the style that was already on the file.

Can you point me to where in this project what you're asking is used, so that I can copy?

Reply:
I'll do the refactor in that case, you also need to extend the tests.

Reply:
@paulomorgado to be honest, it's less work for me to change this than to guide you through your first golang experience. No offence, I just don't have the time for that right now.